atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -152,15 +152,12 @@ public int execute(final AtlasDbServices services) {
     }
 
     private String encodeStartRow(Optional<byte[]> rowBytes) {
-        if (rowBytes.isPresent()) {
-            return BaseEncoding.base16().encode(Arrays.copyOf(rowBytes.get(), 12));
-        }
-        return BaseEncoding.base16().encode(FIRST_ROW);
+        return BaseEncoding.base16().encode(rowBytes.or(FIRST_ROW));
     }
 
     private String encodeEndRow(Optional<byte[]> rowBytes) {
         if (rowBytes.isPresent() && !rowBytes.equals(FIRST_ROW)) {
-            return BaseEncoding.base16().encode(Arrays.copyOf(rowBytes.get(), 12));
+            return BaseEncoding.base16().encode(rowBytes.get());
         } else {
             return BaseEncoding.base16().encode(LAST_ROW);
         }
```

Comment:
in my limited tests this seems safe... any particular concerns? @tmgordeeva @clockfort